.github/workflows/nightly-gcp-experiment-pipeline.yml

```
@@ -5,16 +5,14 @@ on:
     - cron: "30 22 * * *" # Daily 02:30 AM in midnight
 
 jobs:
-  GCP_VM_Instance_Stop:
+  GCP_VM_Instance_Stop_by_Name_And_Label:
     runs-on: ubuntu-latest
-    env:
+    env: 
```

```suggestion
    env:
```
Extra space was added here, YAML files are strict about whitespaces.